src/modules/Plans/views/details/utils/hasPlanMappingsChanged.ts

```diff
@@ -20,15 +20,15 @@ const hasPlanMappingsNetworkChanged = (
   }
 
   // Compare each mapping entry
-  for (const index in currPlanNetworkMaps) {
+  for (let i = 0; i < currPlanNetworkMaps.length; i += 1) {
```

Comment:
Which rule complains about changing for in " to "/of in this for loop?

Reply:
```
For-in loops over arrays skips holes, returns indices as strings, and may visit the prototype chain or other enumerable properties. Use a more robust iteration method such as for-of or array.forEach instead.
```

[@typescript-eslint/no-for-in-array](https://typescript-eslint.io/rules/no-for-in-array)

```
The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.
```

[guard-for-in](https://eslint.org/docs/latest/rules/guard-for-in)